### DISCUSSION TITLE
EXAMPLE: How can I ask a good question?

### DISCUSSION THREAD

**Discussion by mbosselmann** [neuefische/questions#738] [Guides & FAQ]
**This is an example post on how a question should be posed, how it is answered, and what it looks like when an answer is marked as "answer".**

Hello,
I am currently working on my `Card` component and I am getting an **error message** that I don't understand. It seems to be related to my Styled Component and is generated by ESLint.

I have committed and pushed my current status. [It is accessible at this link.](#example-link-to-your-repo) The branch is called "card".

**Description of the error:**
**How can one reproduce the error?**
The error message is displayed in the browser immediately upon starting the Development Server.
**Where and when does it occur?**
Right when starting the Development Server.
**What is expected, what happens?**
I expect my component to render without an error message. If I dismiss the error message, it still appears, but when I restart the Development Server or refresh the browser, the error message comes up again.
**What are the error messages?**

```
ERROR

[eslint]
src/components/Card/Card.js
Line 18:17: 'styled' is not defined no-undef

Search for the keywords to learn more about each error.
```

**What have I already tried:**
I have already searched online for this error and I tried rewriting the Styled Component because I thought I made a syntax error. However, this error keeps coming up. 😢

Thank you for your help!

**Comment by julialoeschel** [comment]
Dear Mareike,

The problem is, that you missd out to import `styled`;
Try to set up the import as: 

`import styled from 'styled-components';`

hope that helps.
Julia

**Reply by mbosselmann** [reply]
Dear Julia,
Thank you for your response!
I did that and the error is now fixed. :)
With that, my problem is resolved. ✅
